### ISSUE TITLE
homepage visualisation

### ISSUE THREAD

**Issue by eodash-issues** [eurodatacube/eodash#13]
### Environment
**Operating system:** MacOS

**Affected browser:** Safari 12

### Steps to reproduce:
when opening the dashboard the landing page is not working properly - split in two layers above each other  which is not easy to navigate, only once one uses in full screen mode the two layers merge into one.

### Expected Behaviour:
one layer

### Actual Behaviour:
see above

### Comments:
No comments
